### PR TITLE
User can view and edit Village Profile

### PR DESCRIPTION
**Pull request by yirano**
Headmaster can view and edit their Village Profile.
image
image

Trello board: https://trello.com/c/0agzcU3T

### CODE REVIEW

File: src/components/Routes.js [hunk 1]
```diff
@@ -31,6 +37,10 @@ function Routes() {
         component={SessionDetails}
       />
       {/* <Route exact path="/donate/" component={Donation} /> */}
+      <Route exact path="/village" component={Village} />
```
Comment (yirano) [inline src/components/Routes.js:40]: Will be changed to `<PrivateRoute>` in future iteration

Comment (ferror18) [reply src/components/Routes.js:40]: Good this is exactly what I was going to point out.

File: src/components/Routes.js [hunk 2]
```diff
@@ -14,6 +18,8 @@ import Libraries from './pages/Admin/Libraries';
 function Routes() {
   return (
     <div>
+      {/* <PrivateRoute exact path="/" component={Dashboard} /> */}
```
Comment (Jahteo) [inline src/components/Routes.js:21]: Duplicates?

File: src/components/pages/Landing/Landing.js [hunk 3]
```diff
@@ -40,20 +40,20 @@ class Layout extends React.Component {
...
           </a>
-          <a href="/" id="bar-header-link">
-            <h1 id="bar-header" style={{ marginBottom: '-25px' }}>
-              Vbb Portal
-              <span className="badge badge-secondary">NEW</span>
-            </h1>
-          </a>
-          <button
+          {/* <a href="/" id="bar-header-link">
+						<h1 id="bar-header" style={{ marginBottom: "-25px" }}>
```
Comment (Jahteo) [inline src/components/pages/Landing/Landing.js:44]: Did prettier do something weird with the spaces/indent here? & in the next few lines

File: src/components/pages/School/School.component.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from 'react';
```
Comment (Jahteo) [inline src/components/pages/School/School.component.js:1]: Love the School.component.js file naming convention, it'll just need to be made consistent in the page names again. later

File: src/components/pages/Village/VillageForm.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+  ButtonMessage,
+} from '../../common/FormStyle';
+
+const baseURL = 'http://54.158.134.245/api';
+
+const initialState = {
+  headmaster: 'Mr Headmaster',
+  village_contact_name: '',
+  village_contact_phone: '',
+  education_contact: {
```
Comment (ferror18) [inline src/components/pages/Village/VillageForm.js:23]: This base URL should probably be in the .env and be an environment variable once the server is deployed

File: src/components/pages/School/SchoolForm.js [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+};
+
+const SchoolForm = props => {
+  const [formData, setFormData] = useState(initialState);
+  const pathname = useHistory().location.pathname;
+  const params = useParams().schoolId;
+  const [form] = Form.useForm();
+
+  useEffect(() => {
+    // Form will populate only if the URL includes "edit"
```
Comment (Jahteo) [inline src/components/pages/School/SchoolForm.js:37]: I was wondering how to set this up, I love it! this is to reuse the same form for editing and adding

**Review by ferror18 — APPROVED** [review]

**Comment by ferror18** [comment]
ok

**Review by Jahteo — APPROVED** [review]
Love it!

File: src/components/Routes.js (hunk 2 above)
Comment (yirano) [reply src/components/Routes.js:21]: Will have to delete the commented out PrivateRoute

**Pull request merged**